src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -DNDEBUG")
+
+set(LIBSLEEF_SRCS sleefdp.c sleefsp.c)
+
+add_library("${TARGET_LIBSLEEF}" SHARED ${LIBSLEEF_SRCS})
+
+# Always compile sleef with -ffp-contract and log at configuration time
+message(STATUS "Using option `${STRICTMATHFLAG}` to compile libsleef")
+target_compile_options(${TARGET_LIBSLEEF} PUBLIC ${STRICTMATHFLAG})
+
+# Shared library versioning
+set_target_properties(${TARGET_LIBSLEEF} PROPERTIES VERSION ${SLEEF_VERSION_MAJOR}.${SLEEF_VERSION_MINOR} SOVERSION ${SLEEF_SOVERSION})
```

Comment:
The minor version should be 1 now.

This versioning is also important in packaging for debian. I set up a launchpad page so that compiled binary of sleef can be installed to ubuntu systems with apt-get when 3.0 was released. However, it is not working well since I haven't got used to debian packaging system.

https://code.launchpad.net/~shibatch/+archive/ubuntu/sleef

I think we need a good way to manage files under debian directory.